appstore/model.go

```diff
@@ -48,14 +57,23 @@ type (
 		ProductID                 string `json:"product_id"`
 		TransactionID             string `json:"transaction_id"`
 		OriginalTransactionID     string `json:"original_transaction_id"`
-		IsTrialPeriod             string `json:"is_trial_period"`
 		AppItemID                 string `json:"app_item_id"`
 		VersionExternalIdentifier string `json:"version_external_identifier"`
 		WebOrderLineItemID        string `json:"web_order_line_item_id"`
+
+		IsTrialPeriod                  string `json:"is_trial_period"`
+		SubscriptionExpirationIntent   string `json:"expiration_intent"`
```

Comment:
The fields `app_item_id`, `version_external_identifier` are also not existing in `InApp`.
I'm not sure if I can remove them.
Because these fields are all existing in `In-App Purchase Receipt Fields` , refer to https://developer.apple.com/library/content/releasenotes/General/ValidateAppStoreReceipt/Chapters/ReceiptFields.html#//apple_ref/doc/uid/TP40010573-CH106-SW12
The document is very unclearly. How do you think of it, remove or not?